code/__HELPERS/matrices.dm

```diff
@@ -208,3 +208,47 @@ round(cos_inv_third+sqrt3_sin, 0.001), round(cos_inv_third-sqrt3_sin, 0.001), ro
 		for(x in 1 to 4)
 			output[offset+x] = round(A[offset+1]*B[x] + A[offset+2]*B[x+4] + A[offset+3]*B[x+8] + A[offset+4]*B[x+12]+(y==5?B[x+16]:0), 0.001)
 	return output
+
+/atom/proc/shake_animation(var/intensity = 8) //Makes the object visibly shake
```

Comment:
why is this in matrices also `proc/Shake` might be better as it would be consistent with everything else